<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dataset selection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Dataset</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="WidgetDataIOSelectTEDatasetExisting" name="combo_dataset">
     <property name="dataset_type" stdset="0">
      <string>sdg-15-3-1-summary</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="WidgetDataIOSelectTELayerExisting" name="combo_prod">
     <property name="layer_type" stdset="0">
      <string>Land Productivity Dynamics (from JRC)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Productivity degradation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Land cover degradation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="WidgetDataIOSelectTELayerExisting" name="combo_sdg">
     <property name="layer_type" stdset="0">
      <string>SDG 15.3.1 Indicator</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="WidgetDataIOSelectTELayerExisting" name="combo_soil">
     <property name="layer_type" stdset="0">
      <string>Soil organic carbon (degradation)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Soil organic carbon degradation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="WidgetDataIOSelectTELayerExisting" name="combo_lc">
     <property name="layer_type" stdset="0">
      <string>Land cover (degradation)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>SDG 15.3.1 indicator</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetDataIOSelectTEDatasetExisting</class>
   <extends>QComboBox</extends>
   <header>LDMP/data_io</header>
  </customwidget>
  <customwidget>
   <class>WidgetDataIOSelectTELayerExisting</class>
   <extends>QComboBox</extends>
   <header>LDMP/data_io</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
